src/tests/dotIgnore.test.ts

```diff
@@ -6,6 +6,7 @@ describe('dot ignore', () => {
     vol.reset();
   });
 
+  // tslint:disable:mocha-no-side-effect-code
```

Comment:
Why is it that only this file requires this `// tslint:disable:mocha-no-side-effect-code`? Most tests in this codebase have side-effects?

Reply:
The rule seems to produce false positives. I don't think there is any issue with the code. `const` protects from unexpected side-effects.